build.rb

```diff
@@ -86,9 +88,20 @@
 end
 
 if params["platform"] == "OSX"
-	system("xcodebuild -workspace ./ReplicationAcceptance/ReplicationAcceptance.xcworkspace -scheme 'RA_Tests_OSX' -destination 'platform=OS X' #{replication_options.join(" ")} test | xcpretty -r junit; exit ${PIPESTATUS[0]}")
+
+	system("xcodebuild -workspace CDTDatastore.xcworkspace -scheme 'Tests OSX' -destination 'platform=OS X' #{replication_options.join(" ")} test | xcpretty -r junit ; exit ${PIPESTATUS[0]}")
+
+	if(params["nightly"] == "true")
+		FileUtils.mv("build/reports/junit.xml","build/reports/osx.xml")
```

Comment:
I think you should have a FileUtils.mv after each system() call which does a test run.

Maybe the names could be $params["platform"]_ra.xml and $params["platform"]_unit.xml to distinguish replication acceptance results from unit test results.